benefits/core/models.py

```diff
@@ -235,6 +231,20 @@ def by_id(id):
         logger.debug(f"Get {EligibilityVerifier.__name__} by id: {id}")
         return EligibilityVerifier.objects.get(id=id)
 
+    claims_scheme_override = models.TextField(
+        help_text="The authentication scheme to use (Optional). If blank, defaults to the value in Claims providers",
+        default=None,
+        null=True,
+        blank=True,
+        verbose_name="Claims scheme",
+    )
+
+    @property
+    def claims_scheme(self):
+        if self.claims_scheme_override == "":
```

Comment:
the default value above is `None`, so probably better to check

```python
if not self.claims_scheme_override:
```

Reply:
Yep, this works better than `== ""`